.github/workflows/test.yml

```
@@ -36,10 +38,10 @@ jobs:
 
       - name: Run Forge build
         run: |
-          forge build --sizes
+          forge build --sizes --via-ir
```

Let's not use `--via-ir` here... this is too new to be used in production.
```suggestion
          forge build --sizes
```

---

Check comment above: fixed